### PR TITLE
Add kor translation terminal tutorial #2880

### PR DESCRIPTION
**Pull request by kohwdev**
Added Korean translation for terminal tutorial. Issue #2880

### CODE REVIEW

**Comment by CandidDeer** [comment]
@Syknapse Please let me know if we can merge this. Thx.

**Comment by TenTraicion** [comment]
#2880

File: translations/README/KOREAN.md [hunk 1]
```diff
@@ -1,281 +1,357 @@
...
 
-이제 모든 준비가 완료되었으므로 본격적으로 프로젝트 기여를 위한 작업을 시작해보겠습니다.
+참고: 이 튜토리얼은 GitHub PC를 기반으로 합니다. [터미널을 편하게 사용하신다면 해당 튜토리얼로 이동하세요 (여기를 클릭하세요)](/terminal_tutorial.md)
 
----
+먼저 작업 환경을 설정해봅시다.
+
+1. GitHub 계정에 로그인하세요. 만약 아직 계정이 없으시다면 [GitHub에 가입하세요](https://github.com/join). 원활한 작업 진행을 위해 [GitHub Hello World 튜토리얼](https://guides.github.com/activities/hello-world/)을 읽고 시작하는 것을 추천드립니다.
+2. [GitHub 데스크탑 앱](https://desktop.github.com/)을 다운로드 후 설치하세요.
+   - 커맨드 라인에서 Git을 사용하는 것이 편하다면 GitHub 데스크탑 앱을 설치하지 않으셔도됩니다. [커맨드 라인(CLI) 튜토리얼 링크](/terminal-tutorial.md).
```
Comment (TenTraicion) [inline translations/README/KOREAN.md:101]: this link should forward to korean terminal tutorial.

File: translations/terminal_tutorial/terminal_tutorial.kor.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+
+[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Join our Discord server!')
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This-Project/)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Contribute-To-This-Project/)
+
+---
+
+> ## **공지:**
+>
+> 이 프로젝트의 메인테이너가 되어 도움을 주고 싶으신가요? 관심이 있으시다면 메인테이너 가이드 [maintainer's guide](/maintainer_guide.md) 를 읽고 [Twitter](https://twitter.com/Syknapse) 에서 DM을 보내주세요.
```
Comment (TenTraicion) [inline translations/terminal_tutorial/terminal_tutorial.kor.md:17]: sadly we don't have any korean translation for maintainer guide. if you happend to be do one, don't forget to change this link.

File: translations/terminal_tutorial/terminal_tutorial.kor.md [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+| :----: | :----: | :----: | :----: | :----: |
+| [Ukrainian](/translations/terminal_tutorial/terminal_tutorial.ukr.md) |
+
+> 프로젝트 문서 번역을 하고 싶다면 환영입니다. [`번역 가이드`](/translations/README.md)를 읽고 기여해주세요.
+
+---
+
+### 준비! :)
+
+참고: 만약 커맨드 라인 환경이 익숙하지 않다면 [GitHub Desktop을 사용하는 GUI 튜토리얼](/README.md)을 봐주세요.
```
Comment (TenTraicion) [inline translations/terminal_tutorial/terminal_tutorial.kor.md:96]: this link should forward to korean README translation

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@kohwdev, Thanks for your contribution. Kindly make the required changes and we are good to go.

File: translations/README/KOREAN.md (hunk 1 above)
Comment (kohwdev) [reply translations/README/KOREAN.md]: Updated the link

File: translations/terminal_tutorial/terminal_tutorial.kor.md (hunk 3 above)
Comment (kohwdev) [reply translations/terminal_tutorial/terminal_tutorial.kor.md]: Updated the link

File: translations/terminal_tutorial/terminal_tutorial.kor.md (hunk 2 above)
Comment (kohwdev) [reply translations/terminal_tutorial/terminal_tutorial.kor.md:17]: I'll try to add the Korean translation for maintainer's guide when I get a chance. Thanks!

**Review by TenTraicion — APPROVED** [review]
Great Work

**Pull request merged**